docs/dev_guide/container_startup.md

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
This should definitely be on the docs site, as it is specifically targeting user workflows around application container startup.